frontend/src/components/CenterFlexBoxMedium.js

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Dominic Heil <d.heil@campus.tu-berlin.de>
```

Comment:
I did not author this file. Please change this to your information (also for the other files).